src/client/app/containers/view.mode.ts

```diff
@@ -14,12 +15,17 @@ import { Strings } from '../strings';
     template: `
         <main ngClass="{{theme$|async}} {{host}}">
             <header class="command__bar">
-                <command class="view-mode" [hidden]="isEmpty" [title]="snippet?.name"></command>
+                <command class="view-disable" [title]="snippet?.name"></command>
+                <command *ngIf="openInPlaygroundSupported" class="view-playground" [title]="strings.openInPlayground">
+                    <command [title]="openInHostString" (click)="openInPlayground(false)"></command>
+                    <command [title]="downloadAsHostFileString" (click)="openInPlayground(true)"></command>
+                </command>
             </header>
             <editor [isViewMode]="true"></editor>
             <footer class="command__bar command__bar--condensed">
-                <command id="feedback" [title]="Feedback" icon="Emoji2" (click)="feedback()"></command>
                 <command icon="Color" [title]="theme$|async" (click)="changeTheme()"></command>
+                <command class="view-disable" [title]="urlString"></command>
+                <command *ngIf="isGist" [title]="strings.openInGithub" (click)="openInGithub()"></command>
```

Comment:
For samples, the URL is a raw URL.  It's not worth the extra complication, I don't think